.gitignore

```diff
@@ -43,3 +43,4 @@ lighthouse-mobile-report
 #e2e-tests
 tests/e2e/videos
 tests/e2e/screenshots
+TEST-PRIVATEKEY
```

Comment:
Can we make `SYNPRESS_PRIVATEKEY` another requirement of the `.env` file instead of a dedicated file?

Reply:
Unfortunately not (yet). This took me a while to figure out; my initial setup was to provide the necessary environment variables through the .env file. But that didn't work: synpress kept complaining the environment variables were not set. I dug into Synpress its code and it turns out Synpress basically takes the environment of the calling shell (the first line is #!/usr/bin/env node ) . So that's why I ended up setting the environment in the other most friendly way I could think of: a custom shell script. I will rename TEST-PRIVATEKEY to SYNPRESS_PRIVATEKEY